app/menus/NewDocumentMenu.tsx

```diff
@@ -26,7 +26,7 @@ function NewDocumentMenu() {
       placement="bottom"
     >
       <Button as={Link} to={newDocumentPath()} icon={<PlusIcon />}>
-        {t("New doc")}
+        {t("New draft")}
```

Comment:
@tommoor should translation have been updated here?

Reply:
It will be displayed on the home screen in top right, might be a bit confusing… I would leave as-is for now